kube/kd/deployment.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   labels:
     version: {{.VERSION}}
   annotations:
-    downscaler/downscale-period: 'Mon-Fri 18:00-08:00 Europe/London'
+    downscaler/uptime: {{.UPTIME_PERIOD}}
```

Comment:
Might be better to not have an uptime period at all? We probably don't want it spinning down on us halfway through a late-night script run.

I think the intention of a downscale period was to scale the pods to 0 at some arbitrary late-night point and not automatically start back up again in the morning. (Which would suit us, as we can just `kubectl scale --replicas=1` when we need it, and we need it less than once a day.)

Reply:
what do you think is best? you're right we don't want it turning off on us, but also we have 10 environments we'll basically never use it on. 
downscale-period: 'Mon-Sun 22:00-23:00 Europe/London' ?

Reply:
shall we just bin it off and scale it down? its pence if we forget